department-of-veterans-affairs/caseflow
# One touch hearing

Resolves #7691 

### Description
Adds a checkbox for Judges to mark a (legacy only) case as a one touch hearing. This change propagates to a DECASS record. Since the column in VACOLS was not there yet, this PR also updates the VACOLS schema.

### Testing Plan
1. Go to BVAAABSHIRE's Queue and mark a legacy case as ready for dispatch
1. Click the one touch hearing box
1. Ensure the DECASS record's `de1touch` column is now "Y"
